migrations/sql/V1.50.3__add_case_description.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE shared.case_file
+ADD COLUMN description TEXT;
+
+ALTER TABLE shared.case_file_h
+ADD COLUMN description TEXT;
```

Comment:
My understanding is that the individual columns in the tables all get captured in the `data_after_executed_operation` column of the corresponding history tables. I could be missing something, but if not I think this column isn't needed